domains/authorization/Authorization.API/API.IntegrationTests/API.IntegrationTests/API/GetConsentTests.GivenNoConsentsExist_WhenUserQueriesForConsents_HttpOKEmptyBody.verified.txt

```diff
@@ -0,0 +1 @@
+﻿{}
```

Comment:
Should this be 
```
{
    Result: []
}
```
?

Reply:
Do our other apis' return a result array?

Reply:
Resolving comment with permission from @ckr123